plugins/modules/purefa_pod.py

```diff
@@ -199,74 +207,98 @@
 POD_QUOTA_VERSION = "2.23"
 THROTTLE_VERSION = "2.31"
 MEMBERS_VERSION = "2.36"
+CONTEXT_VERSION = "2.38"
 
 
 def get_pod(module, array):
     """Return Pod or None"""
-    try:
-        return list(
+    api_version = array.get_rest_version()
+    if LooseVersion(CONTEXT_VERSION) <= LooseVersion(api_version):
+        return bool(
```

Comment:
failure cases also provides error response dict, since its not empty dict bool considering it as true which is causing trouble, should be handled differently